### PR TITLE
feat: beta ribbon

### PR DESCRIPTION
**Pull request by jhult**
Resolves Granola-Team/mina-block-explorer#913

image

### CODE REVIEW

**Comment by n1tranquilla** [comment]
Very sharp

**Comment by n1tranquilla** [comment]
@jhult can you post a shot of the mobile view? It seems mobile tests are failing.

**Comment by jhult** [comment]
It hides the [hamburger button](https://en.wikipedia.org/wiki/Hamburger_button). You can still click on it though.

image

image

**Comment by robinbb** [comment]
Shouldn't the hamburger button be in the upper right, anyway? @jhult

**Comment by robinbb** [comment]
@n1tranquilla What do you think of moving the hamburger to upper right? Would be consistent with the non-mobile website view.

**Comment by n1tranquilla** [comment]
@robinbb I won't refuse any PR that moves the menu to the right!

**Comment by jhult** [comment]
Fixed mobile by adding class `max-md:hidden`.

**Comment by n1tranquilla** [comment]
@jhult if you are looking to hide on mobile, the fragment would be `"... hidden md:block ..."`. See tailwind's [mobile-first](https://tailwindcss.com/docs/responsive-design#working-mobile-first) approach

File: src/header/navigation.rs [hunk 1]
```diff
@@ -124,6 +121,7 @@ pub fn Header() -> impl IntoView {
 
     view! {
         <header class="pl-6 pr-6 pt-4 z-10 bg-main-background flex justify-center md:justify-between items-center fixed top-0 left-0 w-screen h-16">
+            <span class="ribbon max-md:hidden">Beta</span>
```
Comment (n1tranquilla) [inline src/header/navigation.rs:124]:
```suggestion
            <span class="ribbon hidden md:block">Beta</span>
```

**Comment by n1tranquilla** [comment]
This is ready to merge, but we should hold off until we have all the blocks

**Comment by robinbb** [comment]
I think we should just merge. We're close enough!

**Pull request merged**